index.bs

```diff
@@ -0,0 +1,221 @@
+Default Highlight: js
+Complain About: accidental-2119 yes
+</pre>
+
+<pre class=anchors>
+urlPrefix: https://dom.spec.whatwg.org/; spec: DOM;
+    type: dfn; url: #interface-element; text: element
+urlPrefix: https://www.w3.org/TR/css-writing-modes-4/; spec: CSS-WRITING-MODES-4;
+    type: dfn; url: #flow-relative; text: flow-relative offset; 
+urlPrefix: https://w3c.github.io/performance-timeline/; spec: PERFORMANCE-TIMELINE-2;
+    type: interface; url: #the-performanceentry-interface; text: PerformanceEntry;
+    type: attribute; for: PerformanceEntry;
+        text: name; url: #dom-performanceentry-name;
+        text: entryType; url: #dom-performanceentry-entrytype;
+        text: startTime; url: #dom-performanceentry-starttime;
+        text: duration; url: #dom-performanceentry-duration;
+    type: dfn; url: #dfn-register-a-performance-entry-type; text: register a performance entry type;
+    type: dfn; url: #dfn-queue-a-performanceentry; text: Queue the PerformanceEntry;
+    type: dfn; url: #getentriesbytype-method-0; text: getEntriesByType;
+    type: dfn; url: #dom-performanceobserver; text: PerformanceObserver;
+    type: dfn; url: #dom-performanceobserverinit-buffered; text: buffered;
+urlPrefix: https://w3c.github.io/resource-timing/; spec: RESOURCE-TIMING;
+    type: dfn; url: #sec-privacy-security; text: statistical fingerprinting;
+urlPrefix: https://w3c.github.io/hr-time/; spec: HR-TIME-2;
+    type: typedef; url: #idl-def-domhighrestimestamp; text: DOMHighResTimeStamp;
+    type: interface; url: #dfn-performance; text: Performance;
+    type: method; for:Performance;
+        text: now(); url: #dom-performance-now;
+    type: dfn; text: current high resolution time; url: #dfn-current-high-resolution-time;
+    type: attribute; for: WindowOrWorkerGlobalScope;
+        text: performance; url: #dom-windoworworkerglobalscope-performance;
+urlPrefix: https://www.w3.org/TR/CSS21/visuren.html; spec: CSS21;
+    type: dfn; url: #viewport; text: viewport; 
+urlPrefix: https://w3c.github.io/paint-timing/; spec: PAINT-TIMING;
+    type: dfn; url: #mark-paint-timing; text: mark paint timing;
+urlPrefix: https://www.w3.org/TR/css-box-3/; spec: CSS-BOX-3;
+    type: dfn; url: #border-box; text: border box; 
+urlPrefix: https://www.w3.org/TR/css-break-3/; spec: CSS-BREAK-3;
+    type: dfn; url: #box-fragment; text: box fragment; 
+
+</pre>
+
+Introduction {#sec-intro}
+=====================
+
+<div class="non-normative">
+
+<em>This section is non-normative.</em>
+
+When the layout of a page is unstable it leads to a very bad user experience. This shifting of DOM elements is often due to content loading asynchronously and displacing other elements on the page. The layout instability metric identifies these unstable pages by measuring the layout shift fraction of each frame on the page, allowing the user to compute an overall instability score for the page.
```

Comment:
This line is too long, which also makes it harder to review. What I do is put each sentence in a separate line (bikeshed will consider it the same paragraph unless there is a blank line in between).

Reply:
Done.